http/oidc/src/main/java/org/wildfly/security/http/oidc/Oidc.java

```diff
@@ -74,6 +75,16 @@ public class Oidc {
     public static final String PARTIAL = "partial/";
     public static final String PASSWORD = "password";
     public static final String PROMPT = "prompt";
+    public static final String REQUEST = "request";
+    public static final String REQUEST_OBJECT_CONTENT_ENCRYPTION_ALGORITHM = "request-object-content-encryption-algorithm";
```

Comment:
Should this have been updated based on https://github.com/wildfly-security/wildfly-elytron/pull/1984#discussion_r1611752300?